Stripe/STPBankSelectionViewController.m

```diff
@@ -0,0 +1,136 @@
+//
+//  STPBankSelectionViewController.m
+//  Stripe
+//
+//  Created by David Estes on 8/9/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import "STPBankSelectionViewController.h"
+
+#import "NSArray+Stripe.h"
+#import "STPAPIClient+Private.h"
+#import "STPColorUtils.h"
+#import "STPCoreTableViewController+Private.h"
+#import "STPDispatchFunctions.h"
+#import "STPImageLibrary+Private.h"
+#import "STPLocalizationUtils.h"
+#import "STPSectionHeaderView.h"
+#import "STPBankSelectionTableViewCell.h"
+#import "STPPaymentMethodParams.h"
+#import "STPPaymentMethodFPXParams.h"
+#import "UIBarButtonItem+Stripe.h"
+#import "UINavigationBar+Stripe_Theme.h"
+#import "UITableViewCell+Stripe_Borders.h"
+#import "UIViewController+Stripe_NavigationItemProxy.h"
+
+static NSString *const STPBankSelectionCellReuseIdentifier = @"STPBankSelectionCellReuseIdentifier";
+
+@interface STPBankSelectionViewController () <UITableViewDataSource, UITableViewDelegate>
+@property (nonatomic) STPAPIClient *apiClient;
+@property (nonatomic) STPBankType bankType;
+@property (nonatomic) STPFPXBankBrand selectedBank;
+@property (nonatomic) STPPaymentConfiguration *configuration;
+@property (nonatomic, weak) UIImageView *imageView;
+@property (nonatomic) STPSectionHeaderView *headerView;
+@property (nonatomic) BOOL loading;
+@end
+
+@implementation STPBankSelectionViewController
+
+- (instancetype)initWithBankType:(STPBankType)bankType {
+    return [self initWithBankType:bankType configuration:[STPPaymentConfiguration sharedConfiguration] theme:[STPTheme defaultTheme]];
+}
+
+- (instancetype)initWithBankType:(STPBankType)bankType
+                   configuration:(STPPaymentConfiguration *)configuration
+                           theme:(STPTheme *)theme {
+    self = [super initWithTheme:theme];
+    if (self) {
+        NSCAssert(bankType == STPBankTypeFPX, @"STPBankSelectionViewController currently only supports FPX.");
+        _bankType = bankType;
+        _configuration = configuration;
+        _selectedBank = STPFPXBankBrandUnknown;
+        _apiClient = [[STPAPIClient alloc] initWithConfiguration:configuration];
+        self.title = STPLocalizedString(@"Bank Account", @"Title for bank account selector");
+    }
+    return self;
+}
+
+- (void)createAndSetupViews {
+    [super createAndSetupViews];
+
+    [self.tableView registerClass:[STPBankSelectionTableViewCell class] forCellReuseIdentifier:STPBankSelectionCellReuseIdentifier];
+
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
+}
+
+- (void)updateAppearance {
+    [super updateAppearance];
+    
+    self.tableView.allowsSelection = YES;
```

Comment:
is this needed?

Reply:
No, `allowsSelection` is the default, and this method was copied from another VC that doesn't seem to need it either. I'll clean that one up too.